gradle/libs.versions.toml

```diff
@@ -93,6 +93,8 @@ androidx-savedstate = { module = "androidx.savedstate:savedstate", version.ref =
 androidx-viewbinding = { module = "androidx.databinding:viewbinding", version.ref = "android-gradle" }
 
 uri = { module = "com.eygraber:uri-kmp", version.ref = "uri" }
+toml = { module = "net.peanuuutz.tomlkt:tomlkt", version = "0.3.5" }
```

Comment:
Found another library that is based on kotlinx.serialization, published to maven central and supports all toml features.